application/view/omeka/admin/user/edit.phtml

```diff
@@ -27,6 +27,7 @@ $formElement = $this->plugin('formElement');
 </fieldset>
 <fieldset id="change-password" class="section">
     <?php echo $this->formCollection($form->get('change-password'), false); ?>
+    <?php echo $this->passwordRequirements(); ?>
```

Comment:
In the default list of password requirements, is there currently a scenario where we have more than the one requirement? Like in this view:

<img width="1552" alt="Screen Shot 2019-03-22 at 2 45 09 PM" src="https://user-images.githubusercontent.com/606764/54846033-532ea900-4cb1-11e9-980e-809f23131e77.png">

If we only ever have the one requirement, I'm not sure if it should be a list.

Reply:
There can be up to 5 requirements listed there.

Reply:
How are those requirements triggered to appear?

Reply:
Whenever `min_length`,  `min_lowercase`, `min_uppercase`, `min_number`, or `min_symbol` are set to something other than `null`.
```  
  'password' => [
        'min_length' => 6,
        'min_lowercase' => null,
        'min_uppercase' => null,
        'min_number' => null,
        'min_symbol' => null,
        'symbol_list' => '`~!@#$%^&*()-=_+[]\{}|;:",./<>?\'',
    ],
```